packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -110,6 +110,7 @@ are supported:
 * `Parallel`
 * `Succeed`
 * `Fail`
```

Comment:
Could you also change all of these to a inline anchor so that they link to the appropriate sections, like this - `[Task](#task)`.